e2e/extension-fixtures.ts

```
@@ -0,0 +1,44 @@
+import { test as base, chromium, type BrowserContext } from "@playwright/test";
+import path from "path";
+
+const pathToExtension = path.resolve(".output/chrome-mv3");
+
+export const test = base.extend<{
+  context: BrowserContext;
+  extensionId: string;
+}>({
+  context: async (
+    // eslint-disable-next-line no-empty-pattern
+    {},
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)